### ISSUE TITLE
Keycloak Partial Export does not work

### ISSUE THREAD

**Issue by SamTV12345** [kilork/keycloak#147]
I have the issue that I want to create a client that extracts most of the resources from a running Keycloak instance. After I found out that the realm endpoint only returns basic things about a realm I discovered the partial export method for a realm. But somehow it returns an optional string and that string is for me always none. I use the master realm in my export so that should be there always. It's Keycloak v26.0.7

**Comment by kilork** [comment]
The working strategy would be probably to sent same request using `curl` and see that happens.

**Comment by SamTV12345** [comment]
👍 I'll debug this on Monday. Every other request just works only this does not work. Using the admin panel and exporting that there works without an issue. I also checked that it is the same endpoint with the same query parameters.

**Comment by kilork** [comment]
I think it is not stopping us from providing here a better solution. We previously added special case for handling `Location` header in empty responses (#119). This is why this response type `Option<String>`. But I think we did this a little bit non transparent. That we should to improve it is to change how threat empty responses. Seems like `OpenAPI` description generated by `keycloak` is quite limited, the empty response could be text, or have location header. I will prepare now alternative solution, it will be in branch for some time, as it breaks existing clients.

**Comment by kilork** [comment]
@SamTV12345 you may try #150 now.

Basically code call looks like that:

```rust
let partial_export: serde_json::Value = keycloak_admin
        .realm_partial_export_post(&realm, None, None)
        .await?
        .into_response()
        .json()
        .await?;
```

**Comment by kilork** [comment]
@SamTV12345 do you have opportunity to check changes?

**Comment by SamTV12345** [comment]
Yeah it's fixed now. I can import the Real Information through my Rust app and save it locally. Thanks :)
